### DISCUSSION TITLE
tauri package vuejs problem

### DISCUSSION THREAD

**Discussion by hanxiaohuivc** [tauri-apps/tauri#2406] [Q&A] [answered]
Hello everyone. I want to package a vuejs project with tauri. If both ‘distDir’ and ‘devPath’ are set to ‘../dist’, the app can be displayed normally after using ‘yarn tauri dev’. But if the'devPath' is set to "http://localhost:8080", the app cannot display the page after using'yarn tauri dev'. However, after starting with "npm run dev" in another command window, use'yarn tauri' After dev', the app can be used normally. In addition, regardless of the ‘devPath’ setting, the exe generated after using ‘yarn tauri build’ can display page information normally. May I ask what caused this?

**Comment by nothingismagick** [comment, marked as answer]
This is by design. If you want HMR in your dev window, you need to start your development server. If you just point the `devPath` at your dist folder, and change something in there, then that change will not propegate to the dev webview window. The `distDir` value is used exclusively at build time.

**Reply by hanxiaohuivc** [reply]
Thank you very much!

**Reply by hanxiaohuivc** [reply]
In addition, I want to ask you one more question. Why don't you need a server to run the app after tauri is packaged?
